biz.aQute.remote/src/aQute/remote/agent/AgentServer.java

```diff
@@ -38,6 +38,7 @@
 import org.osgi.resource.Requirement;
 import org.osgi.resource.dto.CapabilityDTO;
 import org.osgi.resource.dto.RequirementDTO;
+import org.osgi.service.packageadmin.PackageAdmin;
```

Comment:
Remove this and use the fully qualified name every where. The pre Java 9 you cannot stop the compiler from complaining about deprecated types that you import.